addons/project/models/res_partner.py

```diff
@@ -61,3 +61,27 @@ def _create_portal_users(self):
                 'active': True,
             })
         return created_users
+
+    def action_view_tasks(self):
+        self.ensure_one()
+        action = {
+            'type': 'ir.actions.act_window',
+            'res_model': 'project.task',
+            'name': _("%(name)s's Tasks", name=self.name),
+            'context': {
+                'default_partner_id': self.id,
+            },
+            'views': [(self.env.ref('project.view_task_form2').id, 'form')],
+        }
+        if self.task_count <= 1:
+            action['res_id'] = self.task_ids.id
```

Comment:
Shouldn't we do the same for tickets?

Reply:
It's better to keep consistency I agree! I have modified my enterprise PR to have a similar behavior as here:)